### PR TITLE
merge ubuntu analyzer into debianbase analyzer

### PR DESCRIPTION
**Pull request by tomoyamachi**
- Update dependency : update github.com/genuinetools/reg
- Merge ubuntu analyzer into debianbase analyzer
- Add test files : OS Analyzer

### CODE REVIEW

File: analyzer/os/redhatbase/redhatbase_test.go [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+			path: "./testdata/redhat_6",
+			os:   analyzer.OS{Family: os.RedHat, Name: "6.2"},
+		},
+		"Invalid": {
+			path:    "./testdata/not_redhatbase",
+			wantErr: os.AnalyzeOSError,
+		},
+	}
+	a := redhatOSAnalyzer{}
+	for i, v := range tests {
```
Comment (knqyf263) [inline analyzer/os/redhatbase/redhatbase_test.go]: `i` seems to be index. `name` or `testname` is better.

File: analyzer/os/debianbase/debianbase_test.go [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+			path: "./testdata/ubuntu_18",
+			os:   analyzer.OS{Family: os.Ubuntu, Name: "18.04"},
+		},
+		"Invalid": {
+			path:    "./testdata/not_debianbase",
+			wantErr: os.AnalyzeOSError,
+		},
+	}
+	a := debianbaseOSAnalyzer{}
+	for i, v := range tests {
```
Comment (knqyf263) [inline analyzer/os/debianbase/debianbase_test.go]: the same as above

File: analyzer/os/redhatbase/redhatbase_test.go (hunk 1 above)
Comment (tomoyamachi) [reply analyzer/os/redhatbase/redhatbase_test.go]: fixed

File: analyzer/os/debianbase/debianbase_test.go (hunk 2 above)
Comment (tomoyamachi) [reply analyzer/os/debianbase/debianbase_test.go]: fixed

**Pull request merged**